lib/opentracing/span.rb

```diff
@@ -1,25 +1,27 @@
 module OpenTracing
-  # Span represents an OpenTracer Span
+  # Span represents an OpenTracing Span
   #
   # See http://www.opentracing.io for more information.
   class Span
     NOOP_INSTANCE = Span.new.freeze
 
     # Set the name of the operation
+    #
+    # @param [String] name
     def operation_name=(name)
     end
 
     # Span Context
+    #
+    # @return [SpanContext]
     def context
       SpanContext::NOOP_INSTANCE
     end
 
     # Creates a new {Span}
     #
-    # @param tracer [Tracer] the tracer that created this span
-    # @param span_context [SpanContext] the context of the span
     # @return [Span] a new Span
-    def initialize(tracer, span_context)
+    def initialize(*)
```

Comment:
seems to me that this should be removed entirely, agree? (`Span` init is entirely the concern of the `Tracer` impl)